src/Raw2Gltf.cpp

```diff
@@ -76,7 +76,8 @@ struct GLTFData
     std::shared_ptr<BufferViewData> GetAlignedBufferView(BufferData &buffer, const BufferViewData::GL_ArrayType target)
     {
         unsigned long bufferSize = this->binary->size();
-        if ((bufferSize % 4) > 0) {
```

Comment:
I prefer the opening brace on the same line as the conditional. Actually, a lot of this diff is just code style changes. I'd prefer if we eliminated those (in most cases) from the PR.